03-Addressing  control failures/README.md

```diff
@@ -379,3 +380,66 @@ Any control with evaluation result as not passed and,
 >* Attestation may also expire before actual expiry in cases when the attested state for the control doesn't match with current control state.
 
 [Back to top...](README.md#contents)
+
+----------------------------------------------
+
+## Automated control fix
+
+This feature enables users to bulk fix a control and revert the fix to previous state, if required. Follow the below steps to use this feature.
+
+**Note**:
+This is currently in preview and only below controls' fix are at present supported via this feature: 
+- ADO_AgentPool_AuthZ_Restrict_Broader_Group_Access
+- ADO_Build_AuthZ_Restrict_Broader_Group_Access 
+- ADO_Build_DP_Review_Inactive_Build
+- ADO_Feed_AuthZ_Restrict_Broader_Group_Access
+- ADO_Feed_AuthZ_Dont_Grant_BuildSvcAcct_Permission
+- ADO_Project_AuthZ_Restrict_Broader_Group_Access_on_Agentpool
+- ADO_Project_AuthZ_Restrict_Broader_Group_Access_on_Builds 
+- ADO_Project_AuthZ_Restrict_Broader_Group_Access_on_SvcConn
+- ADO_Project_AuthZ_Restrict_Broader_Group_Access_on_VarGrp
+- ADO_Release_AuthZ_Restrict_Broader_Group_Access 
+- ADO_Release_DP_Review_Inactive_Release
+- ADO_ServiceConnection_AuthZ_Restrict_Broader_Group_Access
+- ADO_VariableGroup_AuthZ_Restrict_Broader_Group_Access
+
+### Remediating control - Step by Step
+
+#### 1. Create backup of current control state
+
+To generate backup of the control locally, run below cmdlet:
```

Comment:
To generate backup of the resource config locally, run below cmdlet

Can we change this to above statement?